src/sparql/exact_syn_from_label.ru

```diff
@@ -0,0 +1,11 @@
+# Create an exact synonym copy for all labels
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+INSERT {
+  ?cls oboInOwl:hasExactSynonym ?label .
+}
+WHERE 
+{
+  ?cls rdfs:label ?label .
+}
```

Comment:
I would exclude obsolete terms from having their label added as a synonym.

Reply:
Hmm, alright. That might put this on pause till later this week but I can do that.

Reply:
Hmm, alright. That might put this on pause till later this week but I can do that.

Reply:
@twhetzel The change is now done. I ran the query afterwards and checked an example case, and the synonym was no longer added for the deprecated term I looked at.

Otherwise this and the build have been approved; just let me know if it's still mergable after you take a look.

Reply:
Thanks for the update.